README.md

```diff
@@ -139,6 +139,7 @@ Styles are not added on `import/require()`, but instead on call to `use`/`ref`.
 |**`transform`** |`{Function}`|`false`|Transform/Conditionally load CSS by passing a transform/condition function|
 |**`insertAt`**|`{String}`|`bottom`|Inserts `<style></style>` at the given position|
 |**`insertInto`**|`{String}`|`<head>`|Inserts `<style></style>` into the given position|
+|**`insertBefore`**|`{String}`|``|A query string specifying an element to insert new `<style>` before, required when `insertAt="before"`|

```

Comment:
I disagree with an extra top level option depending on other one. Possible to pass the an `{Object}` to `insertAt` instead `{ type: 'before',  $: '#id' }` (`$` maybe better named `selector`) [Needs discussion]